main.js

```diff
@@ -2,18 +2,21 @@
 
 
 var saveData = function(db, props) {
-    ref = database.ref(db);
+    // Get a reference to the database service
+    var database = firebase.database();
+    var ref = database.ref(db);
     ref.push(props);
 }
 
 var getData = function(db, id) {
-    ref = database.ref(db);
+    var database = firebase.database();
+    var ref = database.ref(db);
     return ref.getById(id)
 }
 
 // Initializes FriendlyChat.
 function FlavourTheserous() {
-  this.checkSetup();
+  //this.checkSetup();
```

Comment:
I don't know; I wasn't working on this version. I'd been working on the react version

https://github.com/dushr/thesilkroute/tree/react